bears/python/PycodestyleBear.py

```diff
@@ -21,9 +23,14 @@ class PycodestyleBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    @staticmethod
```

Comment:
great. removing this is necessary, because the decorator uses `self.section.aspects` , and so `self` is needed.